### PR TITLE
CRUCIAL bugfix regarding user creation

### PR DESCRIPTION
**Pull request by MurrenMods**
why the FUCK are input emails not .toLowerCase()-ed, and why does the entire fucking backend break when you enter an email with a capital letter?

### CODE REVIEW

File: src/routes/admin-portal/users/add/+page.svelte [hunk 1]
```diff
@@ -7,12 +7,13 @@
 
 	/** Add a new user to the database and the auth system */
 	async function AddUser() {
+		let loweredEmail = email.toLowerCase()
```
Comment (RienkF) [inline src/routes/admin-portal/users/add/+page.svelte:10]:
```suggestion
		const loweredEmail = email.toLowerCase()
```
NO LET'S FOR IMMUTABLE VARS!

Comment (MurrenMods) [reply src/routes/admin-portal/users/add/+page.svelte:10]: i really don't care

File: src/routes/admin-portal/users/add/+page.svelte [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
 		const response = await fetch('/api/admin-portal/users', {
 			method: 'POST',
 			headers: {
 				'Content-Type': 'application/json'
 			},
-			body: JSON.stringify({ email, password, name })
+			body: JSON.stringify({ loweredEmail, password, name })
```
Comment (RienkF) [inline src/routes/admin-portal/users/add/+page.svelte:16]:
```suggestion
			body: JSON.stringify({ email: loweredEmail, password, name })
```
My Boiiij you were about to cause another bug with your bugfix

**Review by RienkF — APPROVED** [review]
Fix it and ship it

File: src/routes/admin-portal/users/add/+page.svelte (hunk 1 above)
Comment (RienkF) [reply src/routes/admin-portal/users/add/+page.svelte]: image
es-lint does

**Pull request merged**